test/functional/job_configs.py

```diff
@@ -97,7 +97,7 @@ def __init__(self, config, expected_to_fail, expected_num_subjobs, expected_num_
     teardown_build:
         - echo "Doing build teardown."
         - sleep 1
-        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt)
+        - ALL_SUBJOB_FILES=$(ls $PROJECT_DIR/subjob_file_*.txt || mktemp -t clusterrunner)
```

Comment:
Would it make more sense to add an if check to the next line instead of making a temp file? If we didn't run any subjobs then there is not really anything to verify.